# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_307_redirect.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_307_redirect.js
--- a/netwerk/test/unit/test_307_redirect.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_307_redirect.js	Tue Feb 12 16:08:25 2019 +0000
@@ -45,7 +45,7 @@
   chan.QueryInterface(Ci.nsIUploadChannel).setUploadStream(uploadStream,
                                                            "text/plain",
                                                            -1);
-  chan.asyncOpen2(new ChannelListener(noHeaderStreamObserver, null));
+  chan.asyncOpen(new ChannelListener(noHeaderStreamObserver, null));
 }
 
 function noHeaderStreamObserver(request, buffer)
@@ -59,7 +59,7 @@
       requestBody;
   uploadStream.setData(streamBody, streamBody.length);
   chan.QueryInterface(Ci.nsIUploadChannel).setUploadStream(uploadStream, "", -1);
-  chan.asyncOpen2(new ChannelListener(headerStreamObserver, null));
+  chan.asyncOpen(new ChannelListener(headerStreamObserver, null));
 }
 
 function headerStreamObserver(request, buffer)
@@ -86,6 +86,6 @@
   chan.QueryInterface(Ci.nsIUploadChannel).setUploadStream(uploadStream,
                                                            "text/plain",
                                                            -1);
-  chan.asyncOpen2(new ChannelListener(noRedirectStreamObserver, null));
+  chan.asyncOpen(new ChannelListener(noRedirectStreamObserver, null));
   do_test_pending();
 }